cpanfile

```diff
@@ -9,6 +9,7 @@ requires 'List::Util', '>= 1.45';
 requires 'Log::Log4perl';
 requires 'Moose';
 requires 'MooseX::Log::Log4perl';
+requires 'Proc::Daemon';
```

Comment:
What's this here for?

Reply:
Turns out that this is required by Hive

Reply:
eHive needs it to properly create workers in a LOCAL meadow.